2021-06-01 20:11:53,803 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:43]
2021-06-01 20:12:04,261 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:43]
2021-06-03 11:45:32,098 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:43]
2021-06-03 11:47:04,105 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:43]
2021-06-03 11:47:51,301 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:43]
2021-06-04 22:45:16,170 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:45]
2021-06-04 22:51:35,045 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:45]
2021-06-09 23:32:26,479 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-09 23:33:05,627 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-10 00:13:41,907 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-10 15:15:43,858 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-10 15:15:52,934 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-10 15:16:06,144 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-10 15:16:15,859 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:52]
2021-06-12 21:29:55,284 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:74]
2021-06-16 00:16:45,844 INFO: Microblog startup [in /Users/bartoszdrozd/Python/flask-microblog/app/__init__.py:78]
2021-06-16 00:16:54,222 ERROR: Exception on /auth/login [GET] [in /Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/bartoszdrozd/Python/flask-microblog/app/auth/routes.py", line 28, in login
    return render_template('auth/login.html', title=_('Sign In'), form=form)
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/Users/bartoszdrozd/Python/flask-microblog/app/templates/auth/login.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/Users/bartoszdrozd/Python/flask-microblog/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 21, in block "body"
    {% block navbar %}
  File "/Users/bartoszdrozd/Python/flask-microblog/app/templates/base.html", line 17, in block "navbar"
    <a class="navbar-brand" href="{{ url_for('index') }}">Microblog</a>
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/app.py", line 2216, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/flask/helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "/Users/bartoszdrozd/Python/flask-microblog/venv/lib/python3.9/site-packages/werkzeug/routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'index'. Did you mean 'main.index' instead?
